creation/models.py

```diff
@@ -1,24 +1,28 @@
 from django.db import models
 from django.utils import timezone
 
-class Blog(models.Model):
-    title = models.CharField(max_length=255, default='', blank=True)
-
-    def __str__(self):
-        return "{}".format(self.title)
-
-class User(models.Model):
-    name = models.CharField(max_length=255)
 
 class Event(models.Model):
     title = models.CharField(max_length=255)
-    #admin = models.ForeignKey(User, on_delete=models.SET_NULL, null=True, related_name="event_admin")
+    # admin = models.ForeignKey(User, on_delete=models.SET_NULL, null=True, related_name="event_admin")
     admin = models.CharField(max_length=100);
     description = models.CharField(default="", max_length=500)
     event_type = models.CharField(max_length=100)
-    users = models.ManyToManyField(User, related_name="event_user")
+    # users = models.ManyToManyField(User, related_name="event_user")
     created_date = models.DateTimeField(
             default=timezone.now) 
     access_code = models.CharField(max_length=8)
+	
+    def __str__(self):
+        return "{}: {} ".format(self.title, self.description)
+
+class Group(models.Model):
+    event = models.ForeignKey(Event, on_delete = models.CASCADE)
+    curr_size = models.IntegerField(default=0)
+    max_size = models.IntegerField(default=4)
```

Comment:
Should add for now `users = models.ForeignKey(User, on_delete=models. DO_NOTHING)`.
Also you will have to put `from django.contrib.auth.models import User` at the top of them file.

Do the same thing for the Event model

Reply:
Added the import, but isn't the "many" end of a many-to-one relationship supposed to have the foreign key?

Reply:
Not sure havent worked with database all that much but we need the Group to have multiple user.

Reply:
yeah, that's what I mean. Every user stores the `id` of a Group object to show what group it's part so, and when we call `Group.user_set()` that'll return all the users in a group.